### ISSUE TITLE
a11y axe tests should check if page actually loaded

### ISSUE THREAD

**Issue by jackahl** [kitconcept/kitconcept.intranet#217]
As @davisagli mentioned in https://github.com/kitconcept/kitconcept.intranet/pull/212#pullrequestreview-3186030288:

Currently a11y axe tests do not check whether the page actually loaded. This leads to it checking the 404 page (which is always compliant) when content is missing or the backend not answering. Because of this tests pass when they should actually fail.
